### PR TITLE
Add support for supplying Fargate platform version

### LINKED ISSUE #19: Add ability to supply Fargate platform version
When making use of `boto3` `run_task()`, add support for supplying a Fargate platform version, defaulting to `LATEST` if none is supplied.

### PR DESCRIPTION
**Pull request by hectcastro**
## Overview

Pass through the `platformVersion` keyword argument when the configured `launchType` is `FARGATE`. In addition, default `platformVersion` to `LATEST`.

Closes #19 

### Demo

image

## Testing Instructions

See: https://github.com/open-apparel-registry/open-apparel-registry/pull/1129

### CODE REVIEW

File: ecsmanage/management/commands/ecsmanage.py [hunk 1]
```diff
@@ -170,34 +171,29 @@ def run_task(self, config, task_def_arn, security_group_id, subnet_id, cmd):
             taskDefinition=task_def_arn
         )["taskDefinition"]
 
+        kwargs = {
```
Comment (hectcastro) [inline ecsmanage/management/commands/ecsmanage.py:174]: Using a `kwargs` dictionary here aids in removing the `task_response` code duplication below.

File: setup.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
 from setuptools import setup
 
-setup(use_scm_version=True,)
+setup(
```
Comment (hectcastro) [inline setup.py:3]: Black complained about this during the PR build.

File: ecsmanage/management/commands/ecsmanage.py (hunk 1 above)
Comment (coalkettler) [reply ecsmanage/management/commands/ecsmanage.py:174]: Nice refactor! 🎯

**Review by coalkettler — APPROVED** [review]
Tested this out with OAR, looks good to go 🚢

**Pull request merged**